lib/objects/objectsInMemServerRedis.js

```diff
@@ -88,7 +88,7 @@ class ObjectsInMemoryServer extends ObjectsInMemoryFileDB {
     /**
      * Separate Namespace from ID and return both
      * @param idWithNamespace ID or Array of IDs containing a redis namespace and the real ID
-     * @returns {{namespace: (string); id: (string|string[]); name?: string; isMeta?: boolean}} Object with namespace and the
+     * @returns {{namespace: (string); id: string; name?: string; isMeta?: boolean}} Object with namespace and the
```

Comment:
actually, it could also be an array, but we only ever use the string version. This way, we have less red squiggles in the code.